playground/android/build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.0'
+        classpath 'com.android.tools.build:gradle:2.2.3'
```

Comment:
com.android.tools.build:gradle:2.2.3 is VERY old. Let's please move to 3.0.1 or at least stay at 2.3.0

Reply:
Detox will not work with that gradle plugin version, it requires 3.x.x